src/main/java/io/axoniq/axonserver/connector/query/impl/SubscriptionQueryStream.java

```diff
@@ -99,26 +171,26 @@ public void onNext(SubscriptionQueryResponse value) {
                         value.getCompleteExceptionally().getClientId()
                 );
                 updateBuffer.onError(exception);
-                if (!initialResultFuture.isDone()) {
-                    initialResultFuture.completeExceptionally(exception);
+                ResultStream<QueryResponse> initial = initialResult.get();
+                if (initial != null && !initial.isClosed()) {
+                    initial.close();
```

Comment:
Shouldn't this close the result stream with an exception?

Reply:
This logic mimics what was there before. When an error comes in on the subscription query, we cancel the stream of initial results.
I decided to remove this logic. The fact that the update stream closed with an error doesn't mean the initial response stream should be interrupted.